client/cpp/synnax/channel/channel.h

```diff
@@ -0,0 +1,72 @@
+#include <string>
+#include <utility>
+#include "v1/channel.pb.h"
+#include <grpcpp/grpcpp.h>
+#include "synnax/telem/telem.h"
+#include "freighter/freighter.h"
+
+#pragma once
+
+using namespace Synnax;
+
+using ChannelKey = std::uint32_t;
+
+namespace Channel {
+    typedef Freighter::UnaryClient<
```

Comment:
From a style point of view, I wouldn't indent all of this inside the `namespace Channel` block, especially since *all* the code in this file is in this namespace.

Reply:
Fixed the formatter in my IDE to not indent.